packages/frameworks/src/frameworks.ts

```diff
@@ -1090,44 +1127,6 @@ export const frameworks = [
       },
     ],
   },
-  {
-    name: 'Sapper',
-    slug: 'sapper',
```

Comment:
I don't think we can safely remove from this list, otherwise `front` will break for existing projects when fetching the logo and other metadata.

Reply:
Should we just revert this or do you think we need the whole example back?